config/api.yml

```diff
@@ -4207,22 +4207,19 @@
         :identifier: zone
       - :name: add
```

Comment:
this should be changed to create, add is historically there (and still supported in code), but we need to be consistent with others.

Reply:
Ok, updated.